samples/boards/nrf52/mesh/onoff_level_app/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+set(QEMU_EXTRA_FLAGS -s)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+target_link_libraries(app subsys__bluetooth)
+
+target_sources(app PRIVATE
+	       src/main.c
+	       src/common.c
+	       src/mesh/ble_mesh.c
+               src/mesh/DEVICE_composition.c
```

Comment:
Why the upper case `DEVICE`? Please be consistent (also, file names are generally lower case in Zephyr). Also, it seems the indentation is off here compared to the previous lines.